contivModel.go

```diff
@@ -719,8 +756,6 @@ func CreateAppProfile(obj *AppProfile) error {
 		return errors.New("Invalid object type")
 	}
 
-	saveObj := obj
```

Comment:
Looks like you don't have the latest modelGen. You're effectively taking out a bugfix.